WORKSPACE

```diff
@@ -0,0 +1,29 @@
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository", "new_git_repository")
+
+git_repository(
```

Comment:
Do you need this `git_repository` command in addition to the `load` command above?